index.html

```diff
@@ -8,31 +8,29 @@
 
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
 
     <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0/angular.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/ngInfiniteScroll/1.3.0/ng-infinite-scroll.js"></script>
     <script src="js/app.js"></script>
 
 
-    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.2/html5shiv.min.js"></script>
-      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-    <![endif]-->
   </head>
   <body ng-app="GiphyApp">
-    <div class="container">
+    <div class="container" ng-controller="GiphyCtrl">
       <h1 class="text-center">Welcome to Giphy Search</h1>
 
       <form class="form-inline">
         <div class="form-group">
           <label for="search">Search</label>
-          <input type="text" placeholder="Search for GIFs" id="search" class="form-control">
+          <input type="text" placeholder="Search for GIFs" id="search" class="form-control" ng-model="search" autofocus>
         </div>
-        <button type="submit" class="btn btn-primary">Search</button>
+        <button type="submit" class="btn btn-primary" ng-click="searchGiphy()">Search</button>
       </form>
-
-      <div id="results"></div>
+      <div class="well" id="results" ng-repeat="myvar in giphy track by $index" infinite-scroll="addMore()" infinite-scroll-distance="0">
+      <span>
+          <img ng-src="{{ myvar.images.fixed_height.url }}"> </span>
+      </div>
     </div>
   </body>
 </html>
```

Comment:
Div element to host the infinite scrolling:

    <div class="row" id="results" infinite-scroll="myPagingFunction()" infinite-scroll-distance="3">
        <div class="well col-md-6" ng-repeat="giphy in giphys track by giphy.id">
            <img ng-src="{{giphy.images.fixed_height.url}}" alt="Description" />
            <a ng-href="{{giphy.images.fixed_height.url}}">link</a>
        </div>
    </div>